modules/playgroundxyzBidAdapter.js

```diff
@@ -44,8 +44,8 @@ export const spec = {
     };
 
     const options = {
-      contentType: 'application/json',
-      withCredentials: false
+      contentType: 'text/plain',
```

Comment:
these are the default options in bidderFactory for a POST request. You can just remove the `options` property.
See: https://github.com/prebid/Prebid.js/blob/master/src/adapters/bidderFactory.js#L260